.github/workflows/staging.yml

```diff
@@ -0,0 +1,93 @@
+name: Staging
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
```

Comment:
Switch this off on pull-request and push? This should be triggered only when triggered manually